README.md

```diff
@@ -5,5 +5,22 @@ Summarisation Using ULMFiT Hopefully?
 ## Download Data using Data Handler
 
 ```bash
-    python3 -m src.download --dataset indosum --pretrained indo_lm
+    python3 -m src.download --filename indosum
+    python3 -m src.download --filename indo_lm
+    python3 -m src.download --filename sentencepiece
 ```
+
+## wiki_ulmfit
+
+```python
+    wiki_trainer = WikiTrainer(Path("./data"))
+    wiki_trainer.download_wiki_files()
```

Comment:
Download should precede object initialisation, so that we can whether all dependencies are valid and there during object init.